documentation/cookbook/working-with-existing-databases.markdown

```diff
@@ -44,15 +45,13 @@ Because Propel has both the ability to create XML schema files based on existing
 To do this you would simply:
 
  1. Follow the steps above to create the `schema.xml` file from existing db.
- 2. Then you would change the target database type and specify connection URL for new database in the project's configuration file (see [configuration](/documentation/10-configuration.html) document)
-
- 3. And then run the `sql:build` task to generate the new DDL:
+ 1. Then you would change the target database type and specify connection URL for new database in the project's configuration file (see [configuration](/documentation/10-configuration.html) document)
```

Comment:
http://propelorm.org/documentation/cookbook/working-with-existing-databases.html#creating-an-xml-schema-from-a-db-structure has list issue. GH markdown assume ordered lists to be marked by 1 on each row. Don't know what markdown standart uses jekyll, but tried to change because current markup even broken